include/tesseract/Makefile.am

```diff
@@ -0,0 +1,54 @@
+AM_CPPFLAGS += \
```

Comment:
Is `AM_CPPFLAGS` really needed here?

Reply:
Not sure. You can try to remove it and see if it compiles.
Remove (usr/local)/include/tesseract in your system first.

Reply:
It works without it because no files are compiled there. But I suggest an even more radical modification, namely no `include/tesseract/Makefile.am` at all. Instead use this code:

    diff --git a/Makefile.am b/Makefile.am
    index 4323f8fc..19cd43f8 100644
    --- a/Makefile.am
    +++ b/Makefile.am
    @@ -65,6 +65,24 @@ doc-clean:
     pkgconfigdir = $(libdir)/pkgconfig
     pkgconfig_DATA = tesseract.pc
     
    +pkginclude_HEADERS = include/tesseract/apitypes.h
    +pkginclude_HEADERS += include/tesseract/baseapi.h
    +pkginclude_HEADERS += include/tesseract/capi.h
    +pkginclude_HEADERS += include/tesseract/genericvector.h
    +pkginclude_HEADERS += include/tesseract/helpers.h
    +pkginclude_HEADERS += include/tesseract/ltrresultiterator.h
    +pkginclude_HEADERS += include/tesseract/ocrclass.h
    +pkginclude_HEADERS += include/tesseract/osdetect.h
    +pkginclude_HEADERS += include/tesseract/pageiterator.h
    +pkginclude_HEADERS += include/tesseract/platform.h
    +pkginclude_HEADERS += include/tesseract/publictypes.h
    +pkginclude_HEADERS += include/tesseract/renderer.h
    +pkginclude_HEADERS += include/tesseract/resultiterator.h
    +pkginclude_HEADERS += include/tesseract/serialis.h
    +pkginclude_HEADERS += include/tesseract/strngs.h
    +pkginclude_HEADERS += include/tesseract/thresholder.h
    +pkginclude_HEADERS += include/tesseract/unichar.h
    +
     # fuzzer-api is used for fuzzing tests.
     # They are run by OSS-Fuzz https://oss-fuzz.com/, but can also be run locally.
     # Note: -fsanitize=fuzzer currently requires the clang++ compiler.

Reply:
Yes, I agree.